cli/cmd/configure_bridge.go

```diff
@@ -0,0 +1,112 @@
+
+const actionExpose = "expose"
+const actionLockdown = "lockdown"
+
+var bridgeCmd = &cobra.Command{
+	Use:   "bridge --action=[expose|lockdown]",
+	Short: "Exposes or locks down the bridge",
+	Long: `Exposes or locks down the Keptn's bridge.
+
+Example:
+	keptn configure bridge --action=expose
+	keptn configure bridge --action=lockdown`,
+	SilenceUsage: true,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		return verifyConfigureBridgeParams(configureBridgeParams)
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		endpoint, apiToken, err := credentialmanager.NewCredentialManager().GetCreds()
+		if err != nil {
+			return errors.New(authErrorMsg)
+		}
+
+		configureBridgeEndpoint := endpoint.Scheme + "://" + endpoint.Host + "/v1/configure/bridge/expose"
+		return configureBridge(configureBridgeEndpoint, apiToken, configureBridgeParams)
+	},
+}
+
+func configureBridge(endpoint string, apiToken string, configureBridgeParams *configureBridgeCmdParams) error {
+	doExpose := *configureBridgeParams.Action == "expose"
+	payload := strconv.FormatBool(doExpose)
+	client := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+			DialContext:     keptnutils.ResolveXipIoWithContext,
+		},
+	}
+	req, err := http.NewRequest("POST", endpoint, bytes.NewReader([]byte(payload)))
+	req.Header.Add("x-token", apiToken)
+	req.Header.Add("content-type", "application/json")
+
+	resp, err := client.Do(req)
+	if err != nil {
+		fmt.Println("Could not complete command: " + err.Error())
+		return err
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+
+	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
+		return errors.New("Could not complete command: " + string(body))
+	}
+
+	if doExpose {
+		if err != nil {
+			return errors.New("Could not " + *configureBridgeParams.Action + " bridge: " + err.Error())
+		}
+		fmt.Printf("Bridge exposed successfully. You can reach it here: https://%s", body)
+	} else {
+		if err != nil {
+			return errors.New("Could not " + *configureBridgeParams.Action + " bridge: " + err.Error())
+		}
+		fmt.Printf("Bridge locked down successfully. Disabled public access.")
+	}
+	return nil
+}
+
+func verifyConfigureBridgeParams(configureBridgeParams *configureBridgeCmdParams) error {
+	if configureBridgeParams.Action == nil {
+		return errors.New("Missing flag 'action'. Must provide either '--action=expose' or '--action=lockdown'")
+	}
+	if *configureBridgeParams.Action != actionExpose && *configureBridgeParams.Action != actionLockdown {
+		return errors.New("Invalid value " + *configureBridgeParams.Action + " 'action'. Must provide either '--action=expose' or '--action=lockdown'")
+	}
+	return nil
+}
+
+func init() {
+	configureCmd.AddCommand(bridgeCmd)
+	configureBridgeParams = &configureBridgeCmdParams{}
+
+	configureBridgeParams.Action = bridgeCmd.Flags().StringP("action", "a", "", "The action to perform [expose,lockdown]")
```

Comment:
Can you please mark the action flag as required?
`configureCmd.MarkFlagRequired("action")`